finance/sc-im/Portfile

```diff
@@ -0,0 +1,27 @@
+PortSystem          1.0
+name                sc-im
+version             0.8.2
+license             BSD
+maintainers         nkorth
+
+categories          finance
+description         Spreadsheet Calculator Improvised
+long_description    An ncurses spreadsheet program for terminal
+
+platforms           darwin
+
+homepage            https://github.com/andmarti1424/sc-im
+master_sites        https://github.com/andmarti1424/sc-im/archive
+distfiles           v${version}.tar.gz
+
+checksums           rmd160  afabf063feda1800fd07e6ee13b4036c7d5a200f \
+                    sha256  7f00c98601e7f7709431fb4cbb83707c87016a3b015d48e5a7c2f018eff4b7f7 \
+                    size    1619617
+
+depends_build       bin:bison:bison
+depends_lib         port:ncurses
+
+use_configure       no
```

Comment:
since you're not using `configure` one would need to make sure the build is using MacPorts' compiler selection and other flags. One way to do this (semi-)automatically would be to include the `makefile` PG, which will attempt to do this (you can then remove the `use_configure` line as that is already set by the PG).